### PR TITLE
Update template.md

### PR DESCRIPTION
**Pull request by chirag-wagento**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [x] Bug fix or improvement
 

## Summary
 Convert to HTML to Markdown.
When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.1/template.md [hunk 1]
```diff
@@ -13,31 +13,31 @@ To get started, edit your local version of this file, using the {% glossarytoolt
...
-  <p>INSERT NOTE TEXT HERE</a>.</p>
+  <p>INSERT NOTE TEXT HERE.</p>
 
 </div>
 
-<h3 id="H3">HEADING 3</h3>
+### HEADING 3 
 PROVIDE TEXT HERE FOR NEXT SECTION.
 
 TO HYPERLINK TO ANOTHER TOPIC... refer to the <a href="{{ page.baseurl }}/extension-dev-guide/bk-extension-dev-guide.html">NAME OF TOPIC OR BOOK</a>.
```
Comment (shrielenee) [inline guides/v2.1/template.md]: Change this link to a markdown link.

File: guides/v2.1/template.md [hunk 2]
```diff
@@ -13,31 +13,31 @@ To get started, edit your local version of this file, using the {% glossarytoolt
...
 
 
 
-<h2 id="H2">HEADING 2</h2>
+## HEADING 2 
 PROVIDE TEXT HERE FOR NEXT SECTION.
 
 ADD DIAGRAM OR ILLUSTRATION <p><img src="{{ site.baseurl }}/common/images/NAME_OF_IMAGEjpg" alt="HOVER TEXT HERE"></p>
```
Comment (shrielenee) [inline guides/v2.1/template.md]: Change this link, and all links (unless they are within an HTML note), to markdown links.

Comment (shrielenee) [inline guides/v2.1/template.md]: ".jpg"

**Comment by chirag-wagento** [comment]
@shrielenee  As per your advice changes link and add .jpg please check

Thanks

**Comment by shrielenee** [comment]
Great job with the changes @chirag-wagento! Thanks!

**Pull request merged**